pyproject.toml

```diff
@@ -1,3 +1,107 @@
+[project]
+name = "b2sdk"
+description = "Backblaze B2 SDK"
+dynamic = ["version"]
+authors = [
+    {name = "Backblaze, Inc", email = "support@backblaze.com"},
+]
+dependencies = [
+    "importlib-metadata>=3.3.0; python_version < '3.8'",
+    "logfury<2.0.0,>=1.0.1",
+    "requests<3.0.0,>=2.9.1",
+    "tqdm<5.0.0,>=4.5.0",
+    "typing-extensions>=4.7.1; python_version < '3.12'",
+    "setuptools>=60.0",
+    "packaging>=21.0",
+]
+requires-python = ">=3.7"
+readme = "README.md"
+license = {text = "MIT"}
```

Comment:
we have our own LICENSE file (which is MIT but with our header, so we probably want to include that)

Reply:
I considered going with `{file = "LICENCE"}` but decided to go with this because:
- The licence would not be quite clear from a glance in PyPI (it would be seen as a custom licence).
- This might affect tools that analyze licenses (like liccheck).
- That's how it was working with `setup.py`, there's no change here.

I mean, the label is not incorrect, this is licensed under MIT, and I suspect that wanting to use other licenses that Backblaze allows is more of a niche that won't be hurt by this label.